src/daos-deserialize/daos-deserialize.c

```diff
@@ -0,0 +1,225 @@
+#include <stdio.h>
+#include <getopt.h>
+#include <string.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <inttypes.h>
+#include <unistd.h>
+#include <errno.h>
+#include <stdbool.h>
+
+/* for daos */
+#ifdef DAOS_SUPPORT
+#include "mfu_daos.h"
+#endif
```

Comment:
@dsikich Since the tool has `daos` in the name, I wonder if we should just remove the `#ifdef DAOS_SUPPORT`?